recipes/muparserx/all/conanfile.py

```diff
@@ -0,0 +1,49 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class MuparserxConan(ConanFile):
+    name = "muparserx"
+    description = "Math Parser Library"
+    license = "BSD"
+    topics = ("conan", "muparserx", "math", "parser")
+    homepage = "https://beltoforion.de/article.php?a=muparserx"
+    url = "https://github.com/beltoforion/muparserx"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": False}
+    generators = "cmake"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder=self._source_subfolder)
+        return cmake
```

Comment:
avoid reconfiguration part 2
```suggestion
        if self._cmake:
            return self._cmake
        self._cmake = CMake(self)
        self._cmake.configure(source_folder=self._source_subfolder)
        self._return cmake
```